clickhouseImport.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-process-env */
```

Comment:
This script imports JSON files to clickhouse? If we want to keep it maybe we should move it to `/post-processing`?

Reply:
Yes, this allows you to dump a previous crawl into Clickhouse if you didn't run the reporter during the actual crawl. I wasn't sure where to put it, but `post-processing` looks to be a good spot.